src/sagemaker/amazon/common.py

```diff
@@ -20,7 +20,7 @@
 
 import numpy as np
 
-from sagemaker.amazon.record_pb2 import Record
+from sagemaker.amazon.record_pb2 import Record  # pylint: disable=E0611
```

Comment:
This can be removed once we upgrade to pylint 3.0 - see https://github.com/pylint-dev/pylint/issues/4987
Reason being that generated .py file is dynamic, and .pyi file has actual class names